src/web/components/CMP.tsx

```diff
@@ -25,11 +21,10 @@ export const CMP = ({ cmpUi }: Props) => {
         }
     }, []);
 
-    const onClose = () => setShow(false);
-
-    if (!show || !cmpUi) {
-        return null;
-    }
-
-    return <ConsentManagementPlatform source="dcr" onClose={onClose} />;
+    return show ? (
+        <ConsentManagementPlatform
+            source="dcr"
+            onClose={() => setShow(false)}
+        />
+    ) : null;
```

Comment:
```suggestion
    if (!show) {
        return null;
    }
   
    return <ConsentManagementPlatform source="dcr" onClose={onClose} />;
```

Reply:
This seems clearer and removes the need for the ternary

Reply:
I didn't like it because of the negation, it became quite confusing when we were discussing the logic in dynamic loading, because you need to make the constant distinction between the value and the if statement.

regarding the ternary, how about this instead? 

```suggestion
    return (
    <>
        {show && 
            <ConsentManagementPlatform
                source="dcr"
                onClose={() => setShow(false)}
            />
        ) }
    </>
```

Reply:
I feel this is more personal preference, so I am happy with whichever. I just had a lot of confusing conversations with people when the code was:
```
if (!show || !cmpUi) {
    return null;
}
```
the double negation made it difficult to discuss. But if its just one variable then it is much more clearer

Reply:
Yes, I agree that double negation with an OR combiner is hard to reason through.

But here we just have one element so it's not complicated. I actually find it a lot clearer.

Reply:
I think the `show && ...` is the one more prevalent in the codebase